docs.html

```diff
@@ -6,6 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="/component_styles/docs.css">
```

Comment:
a comment would be better for readability.

Reply:
sure!